openhcl/underhill_core/src/worker.rs

```diff
@@ -1222,7 +1223,7 @@ async fn new_underhill_vm(
     let boot_info = runtime_params.parsed_openhcl_boot();
 
     // The amount of memory required by the GET igvm_attest request
-    let attestation = get_protocol::IGVM_ATTEST_MSG_SHARED_GPA as u64 * hvdef::HV_PAGE_SIZE;
+    let attestation = get_protocol::IGVM_ATTEST_MSG_MAX_SHARED_GPA as u64 * hvdef::HV_PAGE_SIZE;
```

Comment:
shouldn't this depend on the isolation mode?

Reply:
Will wait for @chris-oo 's change before finalizing the logic here